calico-cloud/compliance/compliance-reports-cis.mdx

```diff
@@ -145,7 +145,7 @@ To manually run a report:
    **Operator**
 
    ```bash
-   curl -O {{filesUrl_CE}}/manifests/compliance-reporter-pod.yaml
+   curl {{filesUrl_CE}}/manifests/compliance-reporter-pod-managed.yaml -o compliance-reporter-pod.yaml
```

Comment:
@bmckercher123 Are these changes needed in this location or should I simply update `calico-cloud_versioned_docs/version-3.17/compliance/compliance-reports-cis.mdx` and `calico-cloud_versioned_docs/version-3.17/compliance/overview.mdx` ?